src/components/Loader/PeopleTable/PersonItem.tsx

```diff
@@ -0,0 +1,56 @@
+import { Link, useParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { Person } from '../../../types';
+
+export const PersonItem = ({ person }: { person: Person }) => {
+  const { slug } = useParams();
```

Comment:
The `useParams` hook is being used inside a component that seems to be a table row for a person. If this component is being rendered as part of a list, it might cause performance issues because `useParams` will be called for each row. Consider lifting the `useParams` hook to a parent component and passing the necessary data down as props.